inc/dg/backend/sparseblockmat_omp_kernels.h

```diff
@@ -4,12 +4,12 @@ namespace dg{
 // multiply kernel
 template<class value_type>
 void ell_multiply_kernel(
-         const value_type* data, const int* cols_idx, const int* data_idx, 
+         const value_type * restrict data, const int * restrict cols_idx, const int * restrict data_idx, 
```

Comment:
unfortunately this does not compile when using gcc (5.4.0) because the restrict keyword is in the C99 standard and not C++. GCC knows __restrict  or similar however